htdocs/core/lib/security.lib.php

```diff
@@ -822,7 +822,7 @@ function checkUserAccessToObject($user, array $featuresarray, $object = 0, $tabl
 		}
 
 		if ($feature == 'payment_sc') {
-			$feature = "chargesociales";
+			$feature = "charges";
```

Comment:
Few lines after, we use "chargesociales". We find also chargesociales for ->element and at different places.

If goal is to fix a problem in permission, i think this must be managed inside the 
user->hasRight method :
Its goal is to be able special values that are not normalized.